hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/config/HiveBigQueryConfig.java

```diff
@@ -110,6 +110,7 @@ public class HiveBigQueryConfig
   public static final String STREAM_FILE_EXTENSION = "stream";
   public static final String JOB_DETAILS_FILE = "job-details.json";
   public static final String QUERY_ID = "bq.connector.query.id";
+  public static final String HIVE_COLUMN_NAME_DELIMITER = ",";
```

Comment:
this should be the property name 'column.name.delimiter', not the value?

Reply:
Good catch, just fixed it